openmrs-client/src/main/java/org/openmrs/mobile/api/workers/UpdateWorker.java

```diff
@@ -0,0 +1,167 @@
+package org.openmrs.mobile.api.workers;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+import android.widget.Switch;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.work.Worker;
+import androidx.work.WorkerParameters;
+import org.openmrs.mobile.api.RestApi;
+import org.openmrs.mobile.api.RestServiceBuilder;
+import org.openmrs.mobile.application.OpenMRSLogger;
+import org.openmrs.mobile.dao.PatientDAO;
+import org.openmrs.mobile.databases.tables.PatientTable;
+import org.openmrs.mobile.listeners.retrofit.DefaultResponseCallbackListener;
+import org.openmrs.mobile.models.Patient;
+import org.openmrs.mobile.models.PatientDto;
+import org.openmrs.mobile.models.PatientPhoto;
+import org.openmrs.mobile.utilities.NetworkUtils;
+import org.openmrs.mobile.utilities.ToastUtil;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class UpdateWorker extends Worker {
+    private static final int ON_SUCCESS = 1;
+    private static final int ON_FAILURE = 2;
+    private static final int ON_SUCCESS_PHOTO_UPDATE = 3;
+    private static final int ON_FAILURE_PHOTO_UPDATE = 4;
+
+    private RestApi restApi;
+    private OpenMRSLogger logger;
+    private Handler mHandler;
+
+    public UpdateWorker(@NonNull Context appContext, @NonNull WorkerParameters workerParams) {
+        super(appContext, workerParams);
+        restApi = RestServiceBuilder.createService(RestApi.class);
+        logger = new OpenMRSLogger();
+
+        mHandler = new Handler(Looper.getMainLooper()) {
+            @Override
+            public void handleMessage(Message msg) {
+                String responseMessage;
+                switch (msg.what) {
+                    case ON_SUCCESS:
+                        String UpdateSuccessPatientName = (String) msg.obj;
```

Comment:
Please follow java conventions. 
```suggestion
                        String updateSuccessPatientName = (String) msg.obj;
```